lab1/README.md

```diff
@@ -0,0 +1,104 @@
+# Lab 1 - Restaurant App
+
+Your goal for this lab is to make an app for a restaurant to manage incoming orders from customers.
+
+You will have 1.5 weeks to complete this lab. **Lab 1 is due at 6:30pm on Thursday, October 13th.** This lab will be done in groups of either two or three. There should be two groups of three.
+
+The following are the minimum requirements for this lab. **You must also add at least one additional feature (minimum one, possibly more!) as well.**
+
+## MVP
+
+Your app should have two “users”, cooks and customers.
+
+### Cooks
+
+- There should be a “Menu” with food items such as Hamburgers, Pizza, etc.
+- Each food item should have an associated ingredients list. (ie, Pizza -> dough, sauce, cheese)
+- Cooks can add/edit/delete food items from the Menu.
+- Cooks can add/edit/delete ingredients from each menu item’s ingredients list.
+- Cooks can view “orders”, which each have an associated customer and food item(s).
+- Cooks can mark orders as completed.
+
+### Customers
+
+- Customers can view the menu and make orders by selecting one or more menu items, and the quantities of each item (i.e., 2 burgers, 4 pizzas, etc.).
+- Customers should be able to specify their name when placing an order.
+
+Your app’s data should be persistent on your phone. This means you should store everything in either an S3 or SQL database so that menu items/orders/ingredients are saved when you exit the app.
+
```

Comment:
People might get confused about how to selectively show users certain screens according to who the user is. Maybe we should explicitly talk about asking the user if they are a cook or a customer. And give examples of how it might look (a simple two button (cook/customet) fragment, login screen, etc. )